app/src/main/java/io/github/droidkaigi/confsched2018/presentation/search/SearchSpeakersFragment.kt

```diff
@@ -71,8 +70,19 @@ class SearchSpeakersFragment : Fragment(), Injectable {
         binding.searchSessionRecycler.apply {
             adapter = groupAdapter
         }
-        binding.searchSessionRecycler.addItemDecoration(DividerItemDecoration(context,
-                LinearLayoutManager(activity).orientation))
+
+        binding.searchSessionRecycler.addItemDecoration(StickyHeaderItemDecoration(context,
+                object : StickyHeaderItemDecoration.Callback {
+                    override fun getGroupId(position: Int): Long {
+                        val initial = speakersSection.getSpeakerNameOrNull(position)?.get(0)
+                        initial ?: return -1
+                        return Character.toUpperCase(initial).toLong()
+                    }
+
+                    override fun getGroupFirstLine(position: Int): String? {
+                        return speakersSection.getSpeakerNameOrNull(position)?.get(0)?.toString()
```

Comment:
@panpanini Thanks! I did not know enough to `Any?.toString()`'s behavior. I changed to safe call on `String?.get(position)` .